articles/cognitive-services/Computer-vision/computer-vision-resource-container-config.md

```diff
@@ -55,7 +55,7 @@ You can specify configuration settings in the optional `ARGS` parameter of the [
 
 For example, the following command instantiates a container from the Recognize Text container image and configures the console logging level to LogLevel.Information, overriding the default configuration setting.
 
-  ```Docker
+  ```DockerFile
```

Comment:
note: this still probably isn't a correct language block because this is actually a block of commands